### PR TITLE
Tck'sforAwsLocalstorageTestAndvirtualfolderImplementation

### PR DESCRIPTION
**Pull request by RohiniKambam**

### CODE REVIEW

**Comment by RohiniKambam** [comment]
virtualfolderawsstoragetest
In this AwsVirtualFolderLocalStorageTest we are creating a folder of cmis:ext_folder in which we are setting cmis:isVirtual as true and checking folder is stored in localStorage or not and simillary we are setting cmis:isVirtual as false and checking folder is stored in localStorage or not
awslocalstoragetest

File: chemistry-opencmis-test-tck/pom.xml [hunk 1]
```diff
@@ -121,5 +122,25 @@
             <version>2.4</version>
             <scope>provided</scope>
         </dependency>
+<!--         <dependency>
```
[the remark this answers is not in the document]
Comment (susmithavani30) [reply chemistry-opencmis-test-tck/pom.xml]: removed

File: chemistry-opencmis-test-tck/pom.xml [hunk 2]
```diff
@@ -1,125 +1,142 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (fahadsuhaib) [inline chemistry-opencmis-test-tck/pom.xml]: Revert changes on this file.

Comment (RohiniKambam) [reply chemistry-opencmis-test-tck/pom.xml]: Updated the pom with latest dependencies
https://github.com/PogeyanOSS/copper-cms/pull/131/commits/437b67d952a291f2e2bf45bf39441a909a52c805

**Pull request merged**